app/services/api_prisoner_checker.rb

```diff
@@ -1,14 +1,7 @@
 class ApiPrisonerChecker
```

Comment:
Are the changes in this class mostly a refactor? It's confusing since this class is unrelated for the location service work.